debian/dist/Debian/rules

```diff
@@ -216,7 +220,9 @@ binary-indep: build install
 	dh_installexamples -i
 	dh_installmenu -i
 	dh_installlogrotate -i
+	dh_systemd_enable -i support-files/mariadb.service support-files/mariadb.socket
 	dh_installinit -i
+	dh_systemd_start -i --restart-after-upgrade mariadb.socket
 	dh_installcron -i 
 	dh_installman -i
 	dh_installinfo -i
```

Comment:
I guess this change (and many other changes) will break packaging on older supported systems with no systemd. But I have no reasonable alternative on my mind, so probably let's do it now and fix later.